### PR TITLE
Add p command for the debugger

### PR DESCRIPTION
**Pull request by gergocs**
Signed-off-by: Gergo Csizi gergocs@inf.u-szeged.hu

### CODE REVIEW

File: tools/debugger/debugger.py [hunk 1]
```diff
@@ -172,6 +172,7 @@ def do_eval(self, args):
         self.debugger.eval(args)
         self.stop = True
     do_e = do_eval
+    do_p = do_eval
```
Comment (zherczeg) [inline tools/debugger/debugger.py:175]: I would also add a `do_print = do_eval` for consistency. A test should be added as well (or an existing is extended).

**Comment by clover2123** [comment]
It would be grateful if you add a TC too.
You can find existing tests under `tools/debugger/tests` directory

File: tools/debugger/tests/do_eval.cmd [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+e b
+e c
+e d
+e e
+print a
+p b
+p c
+p d
+p e
+c
```
Comment (zherczeg) [inline tools/debugger/tests/do_eval.cmd:13]: Add newlines to the end of file.

**Review by zherczeg — COMMENTED** [review]
(informal) LGTM

**Review by ksh8281 — APPROVED** [review]

**Review by clover2123 — APPROVED** [review]

**Pull request merged**